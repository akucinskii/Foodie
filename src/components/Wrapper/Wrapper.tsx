import React from "react";

type Props = {
  children: React.ReactNode;
};
const Wrapper = ({ children }: Props) => {
  return (
    <div
      className="
    flex
    h-full
    w-full
    flex-col
    items-center
    justify-center
    pt-20
    
  "
    >
      {children}
    </div>
  );
};

export default Wrapper;
