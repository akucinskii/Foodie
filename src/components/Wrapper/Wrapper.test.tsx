import { render } from "@testing-library/react";
import { describe, expect, test } from "vitest";
import Wrapper from "./Wrapper";

describe("Wrapper", () => {
  const { container, rerender } = render(<Wrapper>Test</Wrapper>);

  test("should match snapshot", () => {
    expect(container).toMatchSnapshot();
  });

  test("should have children", () => {
    rerender(<Wrapper>Test2</Wrapper>);
    expect(container.textContent).toBe("Test2");
  });
});
