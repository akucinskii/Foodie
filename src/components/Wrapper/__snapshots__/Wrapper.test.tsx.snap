// Vitest Snapshot v1

exports[`Wrapper > should match snapshot 1`] = `
<div>
  <div
    class=" flex h-full w-full flex-col items-center justify-center pt-20 "
  >
    Test
  </div>
</div>
`;
